examples/spark/test.bats

```diff
@@ -31,22 +31,16 @@ setup () {
     spark_config=$spark_dir
     spark_log=/tmp/sparklog
     if [[ $ch_multinode ]]; then
-        # Use the last non-loopback IP address. This is a barely educated
-        # guess and shouldn't be relied on for real code, but hopefully it
-        # works for testing.
-        master_ip=$(  ip -o -f inet addr show \
-                    | grep -F 'scope global' \
-                    | tail -1 \
-                    | sed -r 's/^.+inet ([0-9.]+).+/\1/')
+        master_host="$(hostname)"
```

Comment:
Seems like a comment about what network we are selecting here, and why it's good enough for testing but probably not production, could be useful?

Reply:
I agree, I will add this.